### ISSUE TITLE
Make the installer work with GLIBC older than 2.29

### ISSUE THREAD

**Issue by pirapira** [raiden-network/raiden-wizard#32]
When I tried to run the wizard on xubuntu 18.04 I saw the following messages.  It's complaining about the GLIBC version.

```
skk% chmod u+x raiden_wizard
skk% ./raiden_wizard
[17573] PyInstaller Bootloader 3.x
[17573] LOADER: executable is /home/yoichi/Downloads/home/pirate/Documents/Raiden/Binaries/Wizard/raiden_wizard
[17573] LOADER: homepath is /home/yoichi/Downloads/home/pirate/Documents/Raiden/Binaries/Wizard
[17573] LOADER: _MEIPASS2 is NULL
[17573] LOADER: archivename is /home/yoichi/Downloads/home/pirate/Documents/Raiden/Binaries/Wizard/raiden_wizard
[17573] LOADER: Extracting binaries
[17573] LOADER: Executing self as child
[17573] LOADER: set _MEIPASS2 to /tmp/_MEItw6fNZ
[17573] LOADER: LD_LIBRARY_PATH=/tmp/_MEItw6fNZ
[17573] LOADER: Registering signal handlers
[17574] PyInstaller Bootloader 3.x
[17574] LOADER: executable is /home/yoichi/Downloads/home/pirate/Documents/Raiden/Binaries/Wizard/raiden_wizard
[17574] LOADER: homepath is /home/yoichi/Downloads/home/pirate/Documents/Raiden/Binaries/Wizard
[17574] LOADER: _MEIPASS2 is /tmp/_MEItw6fNZ
[17574] LOADER: archivename is /home/yoichi/Downloads/home/pirate/Documents/Raiden/Binaries/Wizard/raiden_wizard
[17574] LOADER: Already in the child - running user's code.
[17574] LOADER: Python library: /tmp/_MEItw6fNZ/libpython3.7m.so.1.0
[17574] Error loading Python lib '/tmp/_MEItw6fNZ/libpython3.7m.so.1.0': dlopen: /lib/x86_64-linux-gnu/libm.so.6: version `GLIBC_2.29' not found (required by /tmp/_MEItw6fNZ/libpython3.7m.so.1.0)
[17573] LOADER: Restoring signal handlers
[17573] LOADER: freeing args
[17573] LOADER: returning child exit status 255
[17573] LOADER: Back to parent (RC: 255)
[17573] LOADER: Doing cleanup
[17573] LOADER: Freeing archive status for /home/yoichi/Downloads/home/pirate/Documents/Raiden/Binaries/Wizard/raiden_wizard
skk% echo $?                                                                                             ~/Downloads/home/pirate/Documents/Raiden/Binaries/Wizard
255
```

**Comment by pirapira** [comment]
When I upgrade xubuntu to 19.04 the installer works.

**Comment by ulope** [comment]
Related issue in Raiden: raiden-network/raiden#4414

**Comment by fredo** [comment]
we can use debian stretch as raiden does to build the wizard. In general  the wizard should work where RAiden does

**Comment by fredo** [comment]
closed by #249
